src/genie/libs/parser/asa/tests/test_show_route.py

```diff
@@ -5,374 +5,62 @@
+
+    golden_output = {'execute.return_value': """ciscoasa/admin(config)# show route
 
-    golden_output = {'execute.return_value': '''
-        ciscoasa/admin(config)# show route
-         
-        Codes: L - Local, C - connected, S - static, I - IGRP, R - RIP, M - mobile, B - BGP
-        D - EIGRP, E - EGP, EX - EIGRP external, O - OSPF, I - IGRP, IA - OSPF inter area
-        N1 - OSPF NSSA external type 1, N2 - OSPF NSSA external type 2
-        E1 - OSPF external type 1, E2 - OSPF external type 2, E - EGP
-        i - IS-IS, L1 - IS-IS level-1, L2 - IS-IS level-2, ia - IS-IS inter area
-        * - candidate default, su - IS-IS summary, U - per-user static route, o - ODR
-        P - periodic downloaded static route, + - replicated route
-         
-        Gateway of last resort is 10.16.251.1 to network 0.0.0.0
-        
+            Codes: L - Local, C - connected, S - static, I - IGRP, R - RIP, M - mobile, B - BGP
+            D - EIGRP, E - EGP, EX - EIGRP external, O - OSPF, I - IGRP, IA - OSPF inter area
+            N1 - OSPF NSSA external type 1, N2 - OSPF NSSA external type 2
+            E1 - OSPF external type 1, E2 - OSPF external type 2, E - EGP
+            i - IS-IS, L1 - IS-IS level-1, L2 - IS-IS level-2, ia - IS-IS inter area
+            * - candidate default, su - IS-IS summary, U - per-user static route, o - ODR
+            P - periodic downloaded static route, + - replicated route
 
-        S* 0.0.0.0 0.0.0.0 [10] via 10.16.251.1, outside
-                           via 10.16.251.2, pod1000
-        S 0.0.0.1 0.0.0.0 [10/5] via 10.16.255.1, outside
-                                via 10.16.255.2, pod1001
-                                via 10.16.255.3, pod1002
-        C 10.10.1.1 255.255.0.0 is directly connected, _internal_loopback
-        C 10.10.1.2 255.255.254.0 is directly connected, outside
-        L 10.10.1.3 255.255.255.255 is directly connected, pod2000
-                                        is directly connected, pod2002
-        V        10.10.1.4 255.255.255.255
-                                connected by VPN (advertised), admin
-        L        10.10.1.5 255.255.255.255 is directly connected, pod2500
-        C        10.10.1.6 255.255.255.0 
-                                is directly connected, pod3000
-          '''}
+            Gateway of last resort is 10.16.251.1 to network 0.0.0.0
 
-    golden_output_2 = {'execute.return_value': '''
```

Comment:
is it accidentally removed?

Reply:
this was just a test, not supposed to be there

Reply:
sure